dispatch-service/src/main/java/de/muenchen/oss/swim/dispatcher/domain/helper/MetadataHelper.java

```diff
@@ -0,0 +1,67 @@
+package de.muenchen.oss.swim.dispatcher.domain.helper;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.muenchen.oss.swim.dispatcher.domain.exception.MetadataException;
+import de.muenchen.oss.swim.dispatcher.domain.model.Metadata;
+import jakarta.validation.constraints.NotNull;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class MetadataHelper {
```

Comment:
Maybe at least include a JavaDoc saying that this class is completely copied from `handler-core` such that it can be kept in sync when there are changes?